.github/ISSUE_TEMPLATE.md

```diff
@@ -1,22 +1,28 @@
 <!--
 Hi there! thank you for discovering and submitting an issue!
 
-Please first tell us a little bit about the environment you're running:
--->
-
-
-- **OS and version used:** <VERSION> <!-- Windows 10, Ubuntu 15.04... -->
+IMPORTANT: DO NOT include any sensitive or private information. Please use your Azure subscription if you need to share any information from your Azure subscription such as connection strings, service names (IoTHub, Provisioning), etc.
```

Comment:
I can't see the lines you add using view.  For this line, can you change it to: "Please use your Azure subscription to file a support ticket if you need to share any information from your Azure subscription such as connection strings, service names (IoTHub, Provisioning), etc."

Reply:
@yzhong94 These show up only when you open an issue or a PR. The text is a comment and will not show up when posting the issue/PR (we don't want _all_ issues to begin with these sentences).